packages/flutter/lib/src/painting/decoration_image.dart

```diff
@@ -142,6 +146,35 @@ class DecorationImage {
   /// calculated by multiplying [scale] with `scale` of the given [ImageProvider].
   final double scale;
 
+  /// If non-null, the value is multiplied with the opacity of each image
+  /// pixel before painting onto the canvas.
+  ///
+  /// This is more efficient than using [Opacity] or [FadeTransition] to
+  /// change the opacity of an image.
+  final double opacity;
+
+  /// Used to set the filterQuality of the image
+  /// Use the "low" quality setting to scale the image, which corresponds to
+  /// bilinear interpolation, rather than the default "none" which corresponds
+  /// to nearest-neighbor.
+  final FilterQuality filterQuality;
+
+  /// Whether the colors of the image are inverted when drawn.
+  ///
+  /// inverting the colors of an image applies a new color filter to the paint.
```

Comment:
nit: `Inverting`

(capitalization)

Reply:
Most of the docs are copied from RawImage. I also noticed those things, but wanted to make them in the separate PR, I guess I will change them in this one then